asv/util.py

```diff
@@ -1266,9 +1266,11 @@ def truncate_float_list(item, digits=5):
 _global_locks = {}
 
 
-def _init_global_locks(lock_dict):
-    """Initialize global locks in a new multiprocessing process"""
+def _init_global_locks(lock_dict, env):
+    """Initialize global locks in a new multiprocessing process
+    Also inherit the base environment even if using a forkserver"""
     _global_locks.update(lock_dict)
+    os.environ.update(env)
```

Comment:
I understand this is the actual fix. I still don't get why it occurs only with Python 3.14. A change in the `multiprocessing` package?